ci/run_tests.sh

```diff
@@ -19,14 +30,22 @@ PYTEST_CMD="pytest -m \"$PATTERN\" -n auto --dist=loadfile -s --strict --duratio
 # Travis does not have have an X server
 if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
     DISPLAY=DISPLAY=:99.0
-    PYTEST_CMD="xvfb-run -e /dev/stdout $PYTEST_CMD"
+    if [ `uname -m` = 'aarch64' ]; then
+        PYTEST_CMD="xvfb-run -e /dev/stdout pytest -m \"$PATTERN\" -s --strict --durations=10 --junitxml=test-data.xml $TEST_ARGS $COVERAGE pandas"
+    else
+        PYTEST_CMD="xvfb-run -e /dev/stdout $PYTEST_CMD"
+    fi
 fi
 
 echo $PYTEST_CMD
-sh -c "$PYTEST_CMD"
+if [ `uname -m` = 'aarch64' ]; then
+   sudo sh -c "$PYTEST_CMD"
+else
+    sh -c "$PYTEST_CMD"
+fi
 
 if [[ "$COVERAGE" && $? == 0 && "$TRAVIS_BRANCH" == "master" ]]; then
     echo "uploading coverage"
-    echo "bash <(curl -s https://codecov.io/bash) -Z -c -f $COVERAGE_FNAME"
-          bash <(curl -s https://codecov.io/bash) -Z -c -f $COVERAGE_FNAME
+    echo "bash <(curl -s https://codecov.io/bash) -Z -c -F $TYPE -f $COVERAGE_FNAME"
```

Comment:
why is this here?

Reply:
this is almost definitely a rebase mixup; this was fixed about a week ago

Reply:
please, restore the original, I think this is from a merge